src/classes/os-utils.ts

```diff
@@ -18,7 +18,7 @@ export default class OsUtils {
    */
   static getCommand = async (cmd: string): Promise<string> => {
     try {
-      const userConfig: UserConfigJson = await OsUtils.readUserConfig();
+      const userConfig: UserConfigJson = await userConfigUtils.readUserConfig();
```

Comment:
Class names should be captitalized
```suggestion
      const userConfig: UserConfigJson = await UserConfigUtils.readUserConfig();
```